src/repl.rs

```diff
@@ -308,6 +310,19 @@ impl ReplState {
                                 //     // FIXME: Never panic. Instead return Continuation::Error when evaluating.
                                 //     ()
                                 // }
+                            } else if s == &":ASSERT-EMITTED" {
+                                let (first, rest) = store.car_cdr(&rest);
+                                let (second, rest) = store.car_cdr(&rest);
+
+                                assert!(rest.is_nil());
+                                let (_, _, _, emitted) = self.clone().eval_expr(first, store);
+                                let (second_evaled, _, _, _) = self.eval_expr(second, store);
+                                let (mut first_emitted, mut rest_emitted) =
+                                    store.car_cdr(&second_evaled);
+                                for elem in emitted {
+                                    assert_eq!(elem, first_emitted);
```

Comment:
It might be worth a more explicit message in case this fails, perhaps including the index of the failure.